lib/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  overrides: [
+    {
+      files: ['jest-*'],
```

Comment:
I think `axe-helper.js` is also only ever used with jest. We could potentially rename it `jest-axe-helper.js` to reflect this, and match this rule.

Reply:
Feels like we should move the `axe` function into **jest-helpers.js**

Because it's a jest helper (??)

Reply:
@domoscargin New commit added: https://github.com/alphagov/govuk-frontend/pull/2849/commits/a4956a62d7ee648297d5d85d956c787b3a70b6a4

Reply:
I refuse to entertain something which MAKES SO MUCH SENSE.

Nice!